test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java

```diff
@@ -628,17 +628,17 @@ public void testDurableDataLogFailRecoveryReadersPaused() throws Exception {
         // Flush DurableLog to Long Term Storage
         flushToStorage(pravegaRunner.segmentStoreRunner.serviceBuilder);
 
+        pravegaRunner.segmentStoreRunner.close(); // Shutdown SegmentStore
+        pravegaRunner.bookKeeperRunner.close(); // Shutdown BookKeeper & ZooKeeper
+        log.info("SegmentStore, BookKeeper & ZooKeeper shutdown");
+
```

Comment:
Just changed the position. So that after we close everything and then instantiate `Storage` as it should typically happen in the use case.

Reply:
It's not. We are simulating data recovery which requires services to be closed and then restarted.